datatorch/api/api.py

```diff
@@ -1,10 +1,12 @@
-import logging, os, cgi, requests, glob
-from pathlib import Path
+import logging, os, requests, glob
 
 from typing import IO, overload, cast
-from urllib import request
-from urllib.parse import urlencode
-import magic
+
```

Comment:
libmagic is not found on all platforms. Fall back to mimetypes which guesses file type by extension. Not reliable always but does the trick for now